cbv/queries.py

```diff
@@ -0,0 +1,110 @@
+import attrs
+
+from cbv import models
+
+
+@attrs.frozen
+class VersionSwitcher:
+    version_name: str
+    other_versions: list["OtherVersion"]
+
+    @attrs.frozen
+    class OtherVersion:
+        name: str
+        url: str
+
+
+@attrs.frozen
+class NavData:
+    modules: list["Module"]
+
+    @attrs.frozen
+    class Module:
+        source_name: str
+        short_name: str
+        classes: list["NavData.Klass"]
+        active: bool
+
+    @attrs.frozen
+    class Klass:
+        name: str
+        url: str
+        active: bool
+
+
+class NavBuilder:
+    def _to_module_data(
+        self,
+        module: models.Module,
+        active_module: models.Module | None,
+        active_klass: models.Klass | None,
+    ) -> "NavData.Module":
+        return NavData.Module(
+            source_name=module.source_name(),
+            short_name=module.short_name(),
+            classes=[
+                NavData.Klass(
+                    name=klass.name,
+                    url=klass.get_absolute_url(),
+                    active=klass == active_klass,
+                )
+                for klass in module.klass_set.all()
+            ],
+            active=module == active_module,
+        )
+
+    def make_version_switcher(
+        self,
+        projectversion: models.ProjectVersion,
```

Comment:
Everywhere else we talk about project version we've used `project_version`, I think it makes sense to keep this consistent with that?

Reply:
Thanks! Done in [0a58ccc](https://github.com/classy-python/ccbv/pull/210/commits/0a58ccc322fc51da617efad304ec0f28210aba5d)